rules/windows/process_creation/proc_creation_win_inline_win_api_access.yml

```diff
@@ -0,0 +1,73 @@
+title: Accessing WinAPI Via CommandLine
+id: ba3f5c1b-6272-4119-9dbd-0bc8d21c2702
+related:
+    - id: 03d83090-8cba-44a0-b02f-0b756a050306
+      type: derived
+status: experimental
+description: Detects the use of WinAPI Functions via the commandline as seen used by threat actors via the tool winapiexec
+author: Nasreddine Bencherchali
+date: 2022/09/06
+references:
+    - https://twitter.com/m417z/status/1566674631788007425
+tags:
+    - attack.execution
+    - attack.t1106
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains:
```

Comment:
While not add a space the WinAPI functions ?
Fp should be allready low  but will remove the filter here

Reply:
Yep. Good idea